frontend/src/components/advanced-auto-filter/advanced-auto-filter.tsx

```diff
@@ -69,7 +69,10 @@ const AdvancedAutoFilter: FC<AdvancedAutoFilterProps> = (props) => {
       }),
     );
 
-    if (isFiltersEmpty(filters)) {
+    if (
+      isFiltersEmpty({ ...filters, ...checkLists }) &&
+      brandDetails.every((detail) => detail.brandId === '')
+    ) {
```

Comment:
Why is the `brandDetails` check isn't a part of the `isFiltersEmpty`?

Reply:
`brandDetails` structure isn't compatible with `isFiltersEmpty`